CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## Unreleased
 
+#### Fixed
```

Comment:
🙌

Reply:
Just noticed our use of header here is inconsistent... I copied the prior which was using `####`, but now see most (but not all!) are using `###`